src/System.Management.Automation/engine/pipeline.cs

```diff
@@ -257,6 +261,33 @@ private void LogToEventLog()
         /// <exception cref="ObjectDisposedException"></exception>
         internal int Add(CommandProcessorBase commandProcessor)
         {
+            if (ExperimentalFeature.IsEnabled(ExperimentalFeature.PSNativeCommandPreserveBytePipe))
+            {
+                if (commandProcessor is NativeCommandProcessor nativeCommand)
+                {
+                    if (_lastNativeCommand is not null)
+                    {
+                        // Only report experimental feature usage once per pipeline.
+                        if (!_haveReportedNativePipeUsage)
+                        {
+                            ApplicationInsightsTelemetry.SendExperimentalUseData(
+                                ExperimentalFeature.PSNativeCommandPreserveBytePipe,
+                                "native to native created");
```

Comment:
```suggestion
                                "p");
```

Discussed offline with Jim, we decided to go with `f` and `p`